packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -94,6 +101,14 @@ export const RootScreen = () => {
     setIsSplashScreenDismissed(true)
   }, [])
 
+  useEffect(() => {
+    if (isSignUpRedesignEnabled) {
+      if (showHomeStack && startedSignUp) {
+        openWelcomeDrawer()
+      }
+    }
+  }, [isSignUpRedesignEnabled, openWelcomeDrawer, showHomeStack, startedSignUp])
+
```

Comment:
Since we're "redirecting" here, we also have to trigger the modal here 😓 
Not my favorite doing this via a useEffect